database/seeders/SupportTicketSeeder.php

```
@@ -0,0 +1,78 @@
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        $users = User::whereDoesntHave('roles')->get();
+        if ($users->isEmpty()) {
+            $users = User::factory()->count(5)->create();
+        }
+
+        $admin = User::role('admin')->first() ?? $users->first();
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\User::role().


(staticMethod.notFound)